PKGBUILD

```
@@ -46,9 +46,21 @@ package() {
   install -D -m644 99-displaylink.rules "$pkgdir/etc/udev/rules.d/99-displaylink.rules"
   install -D -m755 udev.sh "$pkgdir/opt/displaylink/udev.sh"
 
-  echo "Installing DLM systemd service"
-  install -D -m644 displaylink.service "$pkgdir/usr/lib/systemd/system/displaylink.service"
-  install -D -m755 displaylink-sleep.sh "$pkgdir/usr/lib/systemd/system-sleep/displaylink.sh"
+  echo "Installing DLM runit service"
+
+  mkdir -p "$pkgdir/etc/runit/sv/displaylink"
+  install -Dm 644 displaylink.run "$pkgdir/etc/runit/sv/displaylink/run"
+  chmod u+x "$pkgdir/etc/runit/sv/displaylink/run"
```

```suggestion
  install -Dm744 displaylink.run "$pkgdir/etc/runit/sv/displaylink/run"
```
The `-D` argument already creates parent directories. The -m argument sets permissions, and octal format can do it for all 3 groups. 7 mean read, write and execute. 4 means read. The first digit is user-only.